src/main/java/creativei/entity/Inquiry.java

```diff
@@ -32,17 +29,17 @@ public Inquiry(InquiryVo inquiryVo) throws ParseException {
         this.inquiryStatus=InquiryStatus.stringToEnum(inquiryVo.getInquiryStatus())==null?InquiryStatus.OPEN:InquiryStatus.stringToEnum(inquiryVo.getInquiryStatus());
         this.computerKnowledge=ComputerKnowledge.stringToEnum(inquiryVo.getComputerKnowledge());
         if(inquiryVo.getAddress()!=null)
-            this.inquiryAddress=new InquiryAddress(inquiryVo.getAddress());
+            this.inquiryAddress=new InquiryAddress(inquiryVo.getAddress(),this);
         if(inquiryVo.getEducation()!=null) {
             for(int i=0;i<inquiryVo.getEducation().size();i++) {
-                InquiryEducation inquiryEducations = new InquiryEducation(inquiryVo.getEducation().get(i));
-                this.inquiryEducation.add(inquiryEducations);
+                InquiryEducation inquiryEducation = new InquiryEducation(inquiryVo.getEducation().get(i),this);
+                this.inquiryEducations.add(inquiryEducation);
```

Comment:
Can call new directly from here. No need for extra local reference.

Reply:
issue resolved in commit 057235f